routes/utils.js

```diff
@@ -57,11 +57,15 @@ function deleteFiles(files) {
 	})
 }
 
-function downloadVideo(url, callback) {
-	let videoExtension = url.split('.').pop().toLowerCase();
-	var videoDownloadPath = path.join(__dirname, `video_${Date.now()}_${parseInt(Math.random() * 10000)}` + '.' + videoExtension);
-	var cmd = ("ffmpeg -y -i " + url + " -vcodec copy -acodec copy " + videoDownloadPath);
+function downloadVideo(url, name, callback) {
+	if ( name == null ){
+		name = url;
+	}
+	let videoExtension = name.split('.').pop().toLowerCase();
+	var videoDownloadPath = `video_${Date.now()}_${parseInt(Math.random() * 10000)}` + '.' + videoExtension
```

Comment:
Please change to 
path.join(__dirname, `video_${Date.now()}_${parseInt(Math.random() * 10000)}` + '.' + videoExtension);
Might get issues in production (toolforge).